mutiny-core/src/lib.rs

```diff
@@ -172,6 +175,8 @@ pub enum ActivityItem {
     OnChain(TransactionDetails),
     Lightning(Box<MutinyInvoice>),
     ChannelClosed(ChannelClosure),
+    // /// A payjoin proposal is posted to the directory but not yet broadcast from the sender
+    // PendingPayjoin,
```

Comment:
@benalleng Currently v2 payjoins here are, as you mentioned, handles as on-chain ActivityItems.

This only gets triggered for senders who have signed an original psbt before their payjoin psbt is signed and broadcast by a receiver. We should explore whether or not this is sufficient, what is if not, and document a decision on what to do for the receiver who as of now only will see an ActivityItem once they post a received payjoin psbt that they broadcast